### ISSUE TITLE
bower.json

### ISSUE THREAD

**Issue by johannesjo** [StoryCloud/webkit-assign#3]
Thank you very much for the fix. It would be really cool though, if you could provide also a bower version.

**Comment by jacksonrayhamilton** [comment]
This is a command-line utility, unsuitable for use in browsers. Why would you want this on bower when you could install it via npm?

**Comment by ccamarat** [comment]
He's probably looking for a copy of Angular after it's been run through it. We don't want to get in the business of maintaining third-party libraries, but perhaps we could provide an example of grunt / gulp integration?

**Comment by johannesjo** [comment]
Yes that's it. Sorry I misunderstood how this is supposed to work. Makes sense not to get in the trouble of maintaining third party libraries.

**Comment by jacksonrayhamilton** [comment]
The current programmatic interface is geared towards CLI use. For easier integration with Grunt or Gulp, users will want to specify the output directory and filename. We could update the programmatic interface to return a transform stream which pipes in a file's contents completely and then pipes out patched code.

**Comment by johannesjo** [comment]
This would be really great!

**Comment by jacksonrayhamilton** [comment]
We've added a streams and [gulp](http://gulpjs.com/) interface, and released these in `1.2.0`. You should be able to use these in your build pipeline to automatically fix your code, without needing to maintain separate versions of e.g. `angular.js`.

``` js
var fs = require('fs');
var webkitAssign = require('webkit-assign');
fs.createReadStream('bower_components/angular/angular.js')
    .pipe(webkitAssign())
    .pipe(fs.createWriteStream('build/scripts/angular.js'));
```

``` js
var gulp = require('gulp');
var webkitAssign = require('webkit-assign/gulp');
gulp.src('bower_components/angular/angular.js')
    .pipe(webkitAssign())
    .pipe(gulp.dest('build/scripts'));
```
